.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+<!--
+Describes the nature of your changes. If they are substantial, you should
+further subdivide this into a section describing the problem you are solving and
+another describing your solution.
+-->
+
+## Related issue(s)
+
+<!--
+- Short description of how the PR relates to the issue, including an issue link.
+For example
+- Fixed #100500 by adding lenses to exported items
+
+Write 'None' if there are no related issues (which is discouraged).
+-->
+https://github.com/serokell/tezos-client/issues/
+
+<!--
+Ideally a PR has all of the checkmarks set.
+
+If something in this list is irrelevant to your PR, you should still set this
+checkmark indicating that you are sure it is dealt with (be that by irrelevance).
+
+If you don't set a checkmark (e. g. don't add a test for new functionality),
```

Comment:
> (e. g. don't add a test for new functionality),

Well, now there is no such checkmark :)
Perhaps all this comment can be removed entirely since there is only 1 checkmark.